script/RumpelConfig.sol

```diff
@@ -630,6 +657,248 @@ library RumpelConfig {
         return configs;
     }
 
+    // Mellow Vaults ----
+    function getMellowVaultsGuardProtocolConfigs() internal pure returns (ProtocolGuardConfig[] memory) {
+        ProtocolGuardConfig[] memory configs = new ProtocolGuardConfig[](17);
+
+        // Mellow Simple Lido DV
+        configs[0] = ProtocolGuardConfig({target: MAINNET_MELLOW_DVSTETH, selectorStates: new SelectorState[](2)});
+        configs[0].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[0].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow pzETH
+        configs[1] = ProtocolGuardConfig({target: MAINNET_MELLOW_RENZO_PZETH, selectorStates: new SelectorState[](2)});
+        configs[1].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[1].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow rsENA
+        configs[2] = ProtocolGuardConfig({target: MAINNET_MELLOW_RSENA, selectorStates: new SelectorState[](2)});
```

Comment:
FWIW, my comment on adding deposit assets doesn't apply to this one, since ENA is a reward token!!